api/src/main/java/org/openmrs/api/context/OpenmrsApplicationContextConfig.java

```diff
@@ -0,0 +1,208 @@
+import org.openmrs.annotation.Handler;
+import org.openmrs.annotation.OpenmrsProfileExcludeFilter;
+import org.openmrs.annotation.OpenmrsProfileIncludeFilter;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.CohortService;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.ConditionService;
+import org.openmrs.api.DatatypeService;
+import org.openmrs.api.DiagnosisService;
+import org.openmrs.api.EncounterService;
+import org.openmrs.api.EventListeners;
+import org.openmrs.api.FormService;
+import org.openmrs.api.GlobalPropertyListener;
+import org.openmrs.api.LocationService;
+import org.openmrs.api.MedicationDispenseService;
+import org.openmrs.api.ObsService;
+import org.openmrs.api.OrderService;
+import org.openmrs.api.OrderSetService;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.PersonService;
+import org.openmrs.api.ProgramWorkflowService;
+import org.openmrs.api.ProviderService;
+import org.openmrs.api.SerializationService;
+import org.openmrs.api.UserService;
+import org.openmrs.api.VisitService;
+import org.openmrs.api.db.ContextDAO;
+import org.openmrs.api.db.hibernate.AuditableInterceptor;
+import org.openmrs.api.db.hibernate.DbSessionFactory;
+import org.openmrs.api.db.hibernate.HibernateSessionFactoryBean;
+import org.openmrs.api.impl.AdministrationServiceImpl;
+import org.openmrs.api.impl.GlobalLocaleList;
+import org.openmrs.api.impl.OrderServiceImpl;
+import org.openmrs.api.impl.PersonNameGlobalPropertyListener;
+import org.openmrs.hl7.HL7Service;
+import org.openmrs.logging.LoggingConfigurationGlobalPropertyListener;
+import org.openmrs.messagesource.MessageSourceService;
+import org.openmrs.notification.AlertService;
+import org.openmrs.notification.MessageService;
+import org.openmrs.scheduler.SchedulerService;
+import org.openmrs.util.ConfigUtil;
+import org.openmrs.util.HttpClient;
+import org.openmrs.util.LocaleUtility;
+import org.openmrs.util.LocationUtility;
+import org.openmrs.util.TestTypeFilter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+
+@Configuration
+@ComponentScan(
+    basePackages = "org.openmrs",
+    includeFilters = {
+        @ComponentScan.Filter(type = FilterType.ANNOTATION, classes = Handler.class),
+        @ComponentScan.Filter(type = FilterType.CUSTOM, classes = OpenmrsProfileIncludeFilter.class)
+    },
+    excludeFilters = {
+        @ComponentScan.Filter(type = FilterType.CUSTOM, classes = TestTypeFilter.class),
+        @ComponentScan.Filter(type = FilterType.CUSTOM, classes = OpenmrsProfileExcludeFilter.class)
+    }
+)
+public class OpenmrsApplicationContextConfig {
```

Comment:
There's OpenmrsApplicationContextConfig under org.openmrs.api already. Please add it there instead of creating a new one.